ghstack/config.py

```diff
@@ -119,6 +121,10 @@ def read_config(*, request_circle_token: bool = False) -> Config:  # noqa: C901
             github_username)
         write_back = True
 
+    master_branch = 'master'
+    if config.has_option('ghstack', 'master_branch'):
+        master_branch = config.get('ghstack', 'master_branch')
+
```

Comment:
should we prompt the user?

Reply:
I don't know.  Actually I kind of wonder if this should be a per-repo setting, based on what the default branch for the repo is.